vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/ElementSteps.java

```diff
@@ -280,17 +286,50 @@ public void isElementHasWidthRelativeToTheParentElement(int width)
     }
 
     /**
-     * Clicks on <b>element</b> located by <b>locator</b>
-     * @param locator to locate element
+     * Clicks on an <b>element</b> found by the <b>locator</b>
+     * @param locator The locator to find an element.
+     * @deprecated Use step: "When I click on element located by `$locator`"
      */
+    @Deprecated(since = "0.5.0", forRemoval = true)
     @When("I click on element located `$locator`")
-    public void clickOnElement(Locator locator)
+    public void clickOnElementWithoutRetry(Locator locator)
     {
+        LOGGER.warn("The step: \"When I click on element located `$locator`\" is deprecated and will be removed in "
+                + "VIVIDUS 0.6.0. Use step: \"When I click on element located by `$locator`\"");
         WebElement webElement =
                 baseValidations.assertIfElementExists(String.format(AN_ELEMENT_WITH_ATTRIBUTES, locator), locator);
         mouseActions.click(webElement);
     }
 
+    /**
+     * Clicks on an <b>element</b> found by the <b>locator</b>
+     * <p>The atomic actions performed are:</p>
+     * <ul>
+     * <li>find an element by the locator;</li>
+     * <li>click on the element if it's found, otherwise fail the step;</li>
+     * <li>the first two actions are retried once if <i>StaleElementReferenceException</i> is thrown during their
```

Comment:
you could put a reference or even a link here :) https://www.selenium.dev/exceptions/#stale_element_reference